MapboxNavigation/MGLShape.swift

```diff
@@ -17,12 +17,17 @@ extension MGLPointFeature {
     /**
      Initializes a map point feature representation of the given Turf point feature.
      
-     - parameter pointFeature: The Turf point feature to convert to a map point feature.
+     - parameter pointFeature: The Turf `Point` feature to convert to a map point feature. If the Feature passed is not of type `Point` - initialization fails.
```

Comment:
This is unfortunate, but it’s a consequence of mapbox/mapbox-gl-native#7454 rather than any issue on the navigation side. I wonder if we should add a more generic standalone function that converts between Feature and MGLFeature. But this works for now.